Sources/Web3Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -216,6 +216,54 @@ public class BIP32Keystore: AbstractKeystore {
         try encryptDataToStorage(password, data: serializedRootNode, aesMode: self.keystoreParams!.crypto.cipher)
     }
 
+    /// Fast generation addresses for current account
+    /// used for shows which address user will get when changed number of his wallet
+    /// - Parameters:
+    ///   - password: password of seed storage
+    ///   - number: number of wallets addresses needed to generate from 0 to number-1
+    /// - Returns: Array of addresses generated from 0 to number bound, or empty array in case of error
+    public func getAddressForAccount(password: String, number: Int) -> [EthereumAddress] {
+        guard let decryptedRootNode = try? getPrefixNodeData(password) else {
+            return []
+        }
+        guard let rootNode = HDNode(decryptedRootNode) else {
+            return []
+        }
+        let prefixPath = self.rootPrefix
+        var pathAppendix: String?
+
+        return [Int](0..<number).compactMap({ number in
+            pathAppendix = nil
+            let path = prefixPath + "/\(number)"
+            if path.hasPrefix(prefixPath) {
+                let upperIndex = (path.range(of: prefixPath)?.upperBound)!
+                if upperIndex < path.endIndex {
+                    pathAppendix = String(path[path.index(after: upperIndex)..<path.endIndex])
+                } else {
+                    return nil
+                }
+
+                guard pathAppendix != nil else {
+                    return nil
+                }
+                if pathAppendix!.hasPrefix("/") {
+                    pathAppendix = pathAppendix?.trimmingCharacters(in: CharacterSet.init(charactersIn: "/"))
+                }
+            } else {
+                if path.hasPrefix("/") {
+                    pathAppendix = path.trimmingCharacters(in: CharacterSet.init(charactersIn: "/"))
+                }
+            }
+            guard pathAppendix != nil,
+                  rootNode.depth == prefixPath.components(separatedBy: "/").count - 1,
+                  let newNode = rootNode.derive(path: pathAppendix!, derivePrivateKey: true),
```

Comment:
please no force unwrap

Reply:
Hi, thank you for your comments, but this code in style of all this file
I copied createNewCustomChildAccount method and changed it here, force unwraps had been written here everywhere, that why i used it. And i do not want to change all this file, i used this stile

Reply:
In addition to all of the folks said, I'd like to add that there's a fair point in your arguing about relatively bad code style within all the least file. The thing is that this is how we decided to maintain this lib some time ago: we're leaving all the code that works yet unsound for as long as we're not required to change it. But if we are we're making it as sounded as we can. 

Certainly it's not necessary to rewrite the whole file at the time, actually it's just fine to write in a good way just the required part, but it would be so much better if meanwhile you're applying your effort to the main problem to improve as much relative code as your effort big enough, this would really help to all successor folks who will use this lib afterwards, and you'll get a significant part of that appreciation.

Reply:
I think it is very dangerous to change all this code, because i do not have enough time to understand all logic, i will change this method